dlt/common/storages/load_package.py

```diff
@@ -334,8 +408,13 @@ def create_package(self, load_id: str) -> None:
         self.storage.create_folder(os.path.join(load_id, PackageStorage.COMPLETED_JOBS_FOLDER))
         self.storage.create_folder(os.path.join(load_id, PackageStorage.FAILED_JOBS_FOLDER))
         self.storage.create_folder(os.path.join(load_id, PackageStorage.STARTED_JOBS_FOLDER))
+        # ensure created timestamp is set in state when load package is created
+        state = self.get_load_package_state(load_id)
```

Comment:
we must make this change backward compatible or explicitly upgrade all storages that have load packages internally,

see this test `test_pipeline_with_dlt_update` and similar: they actually upgrade dlt from 0.3 to 0.4 and test packages in process

my worry: people will have old completed load packages or packages in process, they upgrade dlt and it stops working

so you need to test it.